save-demo/src/main/kotlin/com/saveourtool/save/demo/runners/cli/DiktatCliRunner.kt

```diff
@@ -47,7 +47,7 @@ class DiktatCliRunner(
         append(" --disabled_rules=diktat-ruleset:package-naming,standard")
 
         append(" --reporter=plain,output=$outputPath ")
-        if (demoRunRequest.mode == DemoMode.FIX) {
+        if (demoRunRequest.mode == DemoMode.FIX.name) {
```

Comment:
since now `mode` is dynamic, maybe rename `DemoMode` to `DiktatMode`?

Reply:
I guess it will be removed very soon so I see no need in refactoring it right now